<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>761</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QProgressBar" name="durationProgressBar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>460</y>
      <width>381</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSlider" name="durationSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>510</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="clientresources.qrc">
      <normaloff>:/images/icons/pause.png</normaloff>:/images/icons/pause.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>510</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="clientresources.qrc">
      <normaloff>:/images/icons/stop.png</normaloff>:/images/icons/stop.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="muteUnmuteButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>410</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mute</string>
    </property>
   </widget>
   <widget class="QLabel" name="filePathText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select File Path</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>410</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Volume Level</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>510</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="clientresources.qrc">
      <normaloff>:/images/icons/play.png</normaloff>:/images/icons/play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="backwardButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>510</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="clientresources.qrc">
      <normaloff>:/images/icons/rewind.png</normaloff>:/images/icons/rewind.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="forwardButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>510</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="clientresources.qrc">
      <normaloff>:/images/icons/forward.png</normaloff>:/images/icons/forward.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMedia">
    <property name="title">
     <string>Media</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionOpenNetworkStream"/>
   </widget>
   <addaction name="menuMedia"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionOpenNetworkStream">
   <property name="text">
    <string>Open Network Stream</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="clientresources.qrc"/>
 </resources>
 <connections/>
</ui>
